"go install github.com/ipfs/http-api-docs/http-api-docs" command does not work

When I executed the command ``"go install github.com/ipfs/http-api-docs/http-api-docs"``, It did't work,I got errors like this： 

```
endpoints.go:13:2: cannot find package "gx/ipfs/QmXporsyf5xMvffd2eiTDoq85dNpYUynGJhfabzDjwP8uR/go-ipfs/core/commands" in any of:
	/usr/local/opt/go/libexec/src/gx/ipfs/QmXporsyf5xMvffd2eiTDoq85dNpYUynGJhfabzDjwP8uR/go-ipfs/core/commands (from $GOROOT)
	/Users/iclick/projects/gowork/src/gx/ipfs/QmXporsyf5xMvffd2eiTDoq85dNpYUynGJhfabzDjwP8uR/go-ipfs/core/commands (from $GOPATH)
endpoints.go:14:2: cannot find package "gx/ipfs/QmXporsyf5xMvffd2eiTDoq85dNpYUynGJhfabzDjwP8uR/go-ipfs/repo/config" in any of:
	/usr/local/opt/go/libexec/src/gx/ipfs/QmXporsyf5xMvffd2eiTDoq85dNpYUynGJhfabzDjwP8uR/go-ipfs/repo/config (from $GOROOT)
	/Users/iclick/projects/gowork/src/gx/ipfs/QmXporsyf5xMvffd2eiTDoq85dNpYUynGJhfabzDjwP8uR/go-ipfs/repo/config (from $GOPATH)
endpoints.go:15:2: cannot find package "gx/ipfs/QmabLouZTZwhfALuBcssPvkzhbYGMb4394huT7HY4LQ6d3/go-ipfs-cmds" in any of:
	/usr/local/opt/go/libexec/src/gx/ipfs/QmabLouZTZwhfALuBcssPvkzhbYGMb4394huT7HY4LQ6d3/go-ipfs-cmds (from $GOROOT)
	/Users/iclick/projects/gowork/src/gx/ipfs/QmabLouZTZwhfALuBcssPvkzhbYGMb4394huT7HY4LQ6d3/go-ipfs-cmds (from $GOPATH)
```

Could anyone help me, thanks very much!

--- a/README.md
+++ b/README.md
@@ -18,23 +18,11 @@
 
 ## Install
 
-In order to build this project, you need to download `go-ipfs` and install its gx dependencies:
+In order to build this project, you need to first clone the repo and then run `make install`:
 
-```
-> go get github.com/ipfs/go-ipfs
-> cd $GOPATH/src/github.com/ipfs/go-ipfs
-> make deps
-```
-
-Then you can:
-
-```
-> go install github.com/ipfs/http-api-docs/http-api-docs
-```
-
-Or alternatively checkout this repository and run:
-
-```
+```sh
+> git clone https://github.com/ipfs/http-api-docs/http-api-docs
+> cd http-api-docs
 > make install
 ```
 

